.gitignore

```diff
@@ -54,4 +54,6 @@ docker-compose.cloud.prod.yml
 agenta-web/src/pages/auth/[[...path]].tsx
 agenta-web/cypress/screenshots/
 agenta-web/cypress/videos/
-.nextjs_cache/
\ No newline at end of file
+.nextjs_cache/
+node_modules
```

Comment:
We already have a .gitignore file under agenta-web where node_modules are in the ignore list

Reply:
will remove